code/game/objects/items/devices/scanners.dm

```diff
@@ -24,10 +24,27 @@ GENE SCANNER
 	righthand_file = 'icons/mob/inhands/misc/devices_righthand.dmi'
 	materials = list(/datum/material/iron=150)
 
+/obj/item/t_scanner/polytool
+	name = "\improper wire polytool"
+	desc = "A familiar beak adorns this tool. It's inner jaw is a two-step mechanism for gripping and clentching things with the precision that of a surgeon -or a bird, at the end of a small but powerful electric motor. Around to the side is a miniturised multitool and T-ray scanner."
+	icon = 'icons/obj/tools.dmi'
+	icon_state = "polytool-t-ray"
+	lefthand_file = 'icons/mob/inhands/equipment/tools_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/equipment/tools_righthand.dmi'
+	materials = list(/datum/material/iron = 350, /datum/material/gold = 250, /datum/material/silver = 250)
+	item_state = "polytool-t-ray"
+	on = TRUE
```

Comment:
Do czego służy ta zmienna?

Reply:
Mother-type T-Raya zaczyna wyłączony, jako, że attack_self odpowiada za zmianę trybu, chcę, żeby polytool miał włączony skaner gdy jest w trybie T-Raya.

Reply:
Chyba rozumiem. Za to możliwe że uda się troszeczkę upiększyć kod Initialize() niżej.
Czy użytkownik powinien mieć możliwość wyłączyć T-scannera z polytoola?

Reply:
Nie, raczej nie. Jak chce mieć wyłączonego T-Raya to wystarcza możliwość zmiany trybu działania